cmd/notary/keys.go

```diff
@@ -361,13 +361,11 @@ func (k *keyCommander) keysRotate(cmd *cobra.Command, args []string) error {
 		rolesToRotate = []string{data.CanonicalSnapshotRole}
 	case data.CanonicalTargetsRole:
 		rolesToRotate = []string{data.CanonicalTargetsRole}
+	case data.CanonicalTimestampRole:
```

Comment:
I think only if we care about backwards compatibility - if we don't, I'm fine with key rotation to require a role.  (Also, does it make sense to allow the user to manage their own timestamp key?  They can really screw themselves over, but if they're managing everything...)